tools/system_libs.py

```diff
@@ -1370,7 +1407,7 @@ class liblsan_rt(SanitizerLibrary):
                       'lsan_common_emscripten.cpp']
 
 
-class libasan_rt(SanitizerLibrary):
+class libasan_rt(SanitizerLibrary, SjLjLibrary): # ASan uses setjmp/longjmp
```

Comment:
Is it just `libasan` that need this variant and not any of the other sanitizer libraries?  

The only reference I can see to longjmp in asan code seems to be `system/lib/compiler-rt/lib/asan/asan_interceptors.cpp` and that file looks like its ifdef'd out for emscripten.. so is this needed?

Reply:
Oh didn't know it was ifdef'd out. I only grepped files.. Thanks, will remove it.